### ISSUE TITLE
Second Server not listening

### ISSUE THREAD

**Issue by TiltedPhosphor** [arkmanager/ark-server-tools#1160]
My First Server with CrystalIsles is working fine, but now I want to add a second map and I did the same thing, like for my first Server. I made a seperate Ark Installation and a new config. I added `configfile_island="/etc/arkmanager/instances/island.cfg"
` in the arkmanager.cfg and changed the Ports in island.cfg and opened them in my router. But when I start the server it says it is online but it is not listed. I tried reinstalling ark and making a new config, but the problem stays. Does somebody has experience with more than 1 Server and would help me finding the problem thats causing it? 
Thank you in advance

**Comment by klightspeed** [comment]
If you have only put the game ports 1 port apart, then the second server will not start successfully.  They need to be at least 2 ports apart.

You will also need to add firewall rules to allow Port, Port+1, and QueryPort through the firewall.

**Comment by TiltedPhosphor** [comment]
I did change the ports 
`CrystalIsles
QueryPort : 25016
Port : 7778

Island
QueryPort: 25018
Port : 7781`

I opened the ports 7778-7779, 7781-7782, 25016-25019 in the router and firewall. But it doesnt work either.

 Now I researched and found out about ark-dockers, which could be helpful in having more than 1 instance, but do I have to make two dockers for each instance and can I integrate my old Server in a docker?
